src/java/org/apache/cassandra/repair/autorepair/FixedSplitTokenRangeSplitter.java

```diff
@@ -45,6 +45,19 @@ public class FixedSplitTokenRangeSplitter implements IAutoRepairTokenRangeSplitt
 {
     private static final Logger logger = LoggerFactory.getLogger(FixedSplitTokenRangeSplitter.class);
 
+    /**
+     * Selecting the default value is tricky. If we select a small number, individual repairs would be heavy.
+     * On the other hand, if we select a large number, too many repair sessions would be created.
+     * <p>
+     * The default value of 64 works well for non-vnodes, but with vnodes, it can result in too many repair sessions
+     * because the total number of repair sessions on a node is calculated as:
+     * <p>
+     *      number_of_subranges * num_token
```

Comment:
This is no longer the case with the change isn't it?    It'll now be max(number_of_subranges, num_token)

Reply:
yeah, updated the comment